source/security/private-endpoints.txt

```diff
@@ -63,33 +47,42 @@ following information:
   you. If you're using the Admin API, you can find this value by
   calling the :admin-api-endpoint:`List VPC Private Endpoint Services
   <operation/adminListPrivateEndpointServiceInfo>` endpoint and
-  then locating the entry for your App's deployment region in the
-  response.
+  then locating the entry for your App's deployment region and the endpoint type
+  you want to create in the response.
 
   .. io-code-block::
 
-      .. input::
-        :language: sh
+    .. input::
+      :language: sh
+
+      curl https://services.cloud.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/security/private_endpoint_service_infos \
+        -H "Authorization: Bearer <access_token>"
+
+    .. output::
+      :language: json
+      :emphasize-lines: 6
 
-        curl https://services.cloud.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/security/private_endpoint_service_infos \
-          -H "Authorization: Bearer <access_token>"
+      [
+        {
+          "cloud_provider_region": "aws-us-east-1",
+          "service_name": "com.amazonaws.vpce.us-east-1.vpce-svc-0718bec2c0bdeab5c",
+          "endpoint_type": "general"
+        },
+        ...
+      ]
 
-      .. output::
-        :language: json
-        :emphasize-lines: 6
+Add a Private Endpoint
+----------------------
 
-        [
-          {
-            "cloud_provider_region": "aws-us-east-1",
-            "name": "eps_baas-pl-prod_us-east-1_cloud",
-            "dns": "*.aws.services.cloud.mongodb.com",
-            "service_name": "com.amazonaws.vpce.us-east-1.vpce-svc-0f12ab34cd56ef789"
-          },
-          ...
-        ]
+After you've done everything in the Before You Begin section on this page,
+you can add a new Private Endpoint. You can do this with the App Services
+UI or the Admin API.
 
-Once you have this information, you can set up the Private Endpoint
-connection using either the App Services UI or the Admin API.
+You can add a new General Endpoint or a new Sync Endpoint. You can also
```

Comment:
(yeah, the definitions should come before we start talking about them like users already know what these mean)